test/fixtures/debug-uncaught-async.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const debug = require('_debugger');
+
+function emit() {
+  const error = new Error('fhqwhgads');
+  process.emit('uncaughtException', error);
+}
+
+assert.doesNotThrow(emit);
+
+debug.start(['sterrance']);
```

Comment:
That's sending `debug.start()` an `argv` array of length 1 to avoid [code that exits if `argv` is empty](https://github.com/nodejs/node/blob/30dbc9d9610e82fc1c7186a79d61e2d06e512d08/lib/_debugger.js#L18-L23).

I'll add a comment and/or change `sterrance` to a more meaningful string.